main.go

```diff
@@ -132,8 +132,8 @@ func main() {
 		os.Exit(1)
 	}
 
-	// Set 'MANUAL_CSR_APPROVAL=true' to disable ByoAdmission controller. Now CSRs should be approved manually.
-	if os.Getenv("MANUAL_CSR_APPROVAL") != "true" {
+	// Set 'MANUAL_CSR_APPROVAL=enable' to disable ByoAdmission controller. Now CSRs should be approved manually.
+	if os.Getenv("MANUAL_CSR_APPROVAL") != "enable" {
```

Comment:
true / false is more intuitive IMO. Any reason for using enable / disable?

Reply:
This change enables users to pass MANUAL_CSR_APPROVAL via clusterctl config. When we use true/false, it fails during conversion from unstructured object to concrete type [here](https://github.com/kubernetes/apimachinery/blob/master/pkg/runtime/converter.go#L363) because it assumes the value true/false as boolean while expected type is string.